### DISCUSSION TITLE
Search the Omnisearch cache on mobile

### DISCUSSION THREAD

**Discussion by carlba** [scambier/obsidian-omnisearch#325] [Q&A] [answered]
I know textextraction does not work on mobile but the documentation indicates that it is possible to sync the cache from desktop to mobile devices,  thus making PDFs searchable on mobile.

When I do try to search for stuff in PDFs on my mobile I get this.

![image](https://github.com/scambier/obsidian-omnisearch/assets/1884191/6c42f858-a212-4769-a659-3b2cd396cea1)

What do I need to do to sync the cache to enable searching in PDFs?

**Comment by scambier** [comment]
Probably a bug, I'll take a look at it

**Reply by scambier** [reply, marked as answer]
Ok I missed your phrase after the screenshot

> What do I need to do to sync the cache to enable searching in PDFs?

There are several solutions you can use to sync your data in Obsidian, see https://help.obsidian.md/Getting+started/Sync+your+notes+across+devices

**Comment by ely-see** [comment]
Hello, 

I have the same issue ( #https://github.com/scambier/obsidian-omnisearch/issues/395), even with Obsidian Sync that syncs the Cache and so - according to the documentation - the PDF must be searchable via the synced Cache.

Any updates on this?

**Reply by scambier** [reply]
You need to have both Omnisearch and Text Extractor installed and enabled on your devices, and you need to sync the folder `.obsidian/plugins/text-extractor/cache` and its contents. Then on your phone, enable indexing of images and PDFs, restart Obsidian, and it should work.

I don't use Obsidian Sync, nor do I own an iOS device, so I don't know if there are some specifics steps to take there.
